lib/python/rose/bush.py

```diff
@@ -111,6 +121,7 @@ def broadcast_events(self, user, suite, form=None):
         """List broadcasts history of a running or completed suite."""
         user_suite_dir = self._get_user_suite_dir(user, suite)
         data = {
+            "title": self.title,
```

Comment:
It feels like these repeated data dictionaries (here and in subsequent methods) feel like they should be able to be cut down to some sort of object with overrides for the settings that can change i.e. we're always using host, and rose version. Probably not an issue for here but I think the addition of title in each case highlights ongoing redundancy.